---
std_msgs/Duration coverage_time
